obspy/signal/tests/test_spectral_estimation.py

```diff
@@ -550,16 +550,17 @@ def test_earthquake_models(self):
         fig = ppsd.plot(show_earthquakes=(1, 4, 5, 15), show=False)
         ax = fig.axes[0]
         for index, magnitude in enumerate([1.5, 2.5, 3.5]):
-            frequencies, power = earthquake_models[(magnitude, 10)]
+            frequencies, power = earthquake_models[(magnitude, 10))
```

Comment:
needs reverting

Reply:
Actually I managed to get pytest to work and there are more issues as there are other lines ... I am fixing it.

Reply:
Changed approach but done? See below